### PR TITLE
Bypass login

### PR DESCRIPTION
**Pull request by gilhsu**
- Create request Bundle for Patient
- Adjust `patientSlice` to `patientDataSlice`
- Add `skipLogin` feature to bypass auth for dev purposes and mocking patient data when landing on Summary Page
- Create local mock Patient Bundle file
- Flatten response Bundle to make Vital Signs and Lab Results top level resources
- Create `fhirReader.js` to parse fhir data
- Create Collections Tab (unrelated but easy, just wanted to sneak it in)

### CODE REVIEW

**Review by vadi2 — APPROVED** [review]

**Comment by vadi2** [comment]
What screen dimensions are you using? 
image

**Comment by gilhsu** [comment]
> What screen dimensions are you using?

I'm using the iphone 11 emulator which appears to be `1792‑by‑828‑pixel resolution at 326 ppi`

I'll quickly modify this to remove some padding at the top. Then I'll work on introducing a library so we can use `%` instead of fix pixels so the app can be more responsive.

**Comment by gilhsu** [comment]
Refactored using `%`. Iphone 8 vs 11

image

**Comment by vadi2** [comment]
That works!

File: src/screens/LoginScreen.js [hunk 1]
```diff
@@ -1,52 +1,69 @@
...
+              resizeMode="contain"
+            />
+            <View style={styles.descriptionContainer}>
+              <Text style={styles.descriptionText}>Discovery Mobile</Text>
+            </View>
           </View>
+          <Login navigation={navigation} />
+        </View>
+        <View style={styles.vermonsterContainer}>
+          <Button title="Skip Login" onPress={handleSkipLogin} />
```
Comment (foxtree) [inline src/screens/LoginScreen.js]: Can this be disabled based on some ENV?   (perhaps development mode -- if deploys are _not_ development mode?)

Comment (gilhsu) [reply src/screens/LoginScreen.js]: +1

Added a `process.env` lookup for development

**Review by foxtree — APPROVED** [review]

**Pull request merged**